types/hummus-recipe/index.d.ts

```diff
@@ -55,7 +55,13 @@ declare namespace Recipe {
         author?: string;
         title?: string;
         subject?: string;
+        colorspace?: string;
```

Comment:
This should be `colorspace?: 'rgb' | 'cmyk' | 'gray';` right?

Reply:
You're right. I've checked tests in hummusRecipe just to make sure: https://github.com/chunyenHuang/hummusRecipe/blob/master/tests/color.js